flows/full_pipeline.py

```diff
@@ -69,7 +70,7 @@ async def create_full_pipeline_summary_artifact(
 async def full_pipeline(
     classifier_specs: Sequence[ClassifierSpec] | None = None,
     document_ids: Sequence[DocumentImportId] | None = None,
-    inference_use_new_and_updated: bool = False,
+    document_ids_s3_uri: S3Uri | None = None,
```

Comment:
thought: Should we change this to an s3 path? We can then convert to an s3 uri within the inference flow.